### ISSUE TITLE
A11y: Connect switches and information text so that the two are linked for screen readers

### ISSUE THREAD

**Issue by daniellekirkwood** [vector-im/element-web#24929]
<img width="526" alt="Screenshot 2023-03-24 at 12 58 53" src="https://user-images.githubusercontent.com/89144281/227527535-d100e24e-d705-4df4-9b5a-c167be4b032c.png">

The designation including the information text of the switch is not linked to
the associated control element (example marked in blue). Therefore, when the
toggle is focused, the caption is not read out by the screen reader, making it difficult
for blind users to access the form.

This applies to all switches in "Settings"

Suggestion: Labels should be surrounded by a label tag, which is linked to the associated control element (example marked in red) using the for attribute and an id.

Please ask @daniellekirkwood about timesheeting

**Comment by NSV1991** [comment]
@daniellekirkwood @gsouquet I like to work on this issue

**Comment by daniellekirkwood** [comment]
Thanks!

**Comment by t3chguy** [comment]
Fixed by https://github.com/matrix-org/matrix-react-sdk/pull/10553
